Add license information for OSS (#265)
anbox-cloud/discourse-docs: ref/license-information.md

Anbox Cloud is built using multiple open source software components. This guide lists those components and where to find their license information.

> | Open source software | Where to find the license information |

Author: Do "software components" and "Where to find their license information" sound better?

> | Open source software | Where to find the license information |

Reviewer: I think so!

|--|--|

> | LXD images | Within the LXD image, <br/>`/usr/share/licenses` and `/opt/mesa/share/licenses` |

Reviewer 2: If people are interested in checking the copyright file for anbox, check the following path 
/usr/share/doc/anbox/copyright

Reviewer 3: Better use `/usr/share/doc/*/copyright`

> | LXD images | Within the LXD image, `/usr/share/licenses` <br/> The copyright information of Anbox Cloud is available at `/usr/share/doc/*/copyright`. |

Reviewer 3: ```suggestion
| LXD images | Within the LXD image, `/usr/share/licenses` and `/usr/share/doc/*/copyright` <br/> |
```
We do not have to call out Anbox specifically but just list all places.

> | LXD images | Within the LXD image, <br/>`/usr/share/licenses` and `/opt/mesa/share/licenses`<br/> The copyright information of Anbox Cloud is available at `/usr/share/doc/*/copyright`. |

Reviewer 2: We're not there yet for mesa licenses until https://warthogs.atlassian.net/browse/AC-1661 get landed. If we're going to publlish this after it's merged, we'd better drop it

| Android | In the Android container, <br/> **Settings** > **About emulated device** > **Legal information** |

> | Snaps | On the host system,<br/>`/snap/$name/current/COPYRIGHT.*` *where $name is the name of the Snap* |

Author: These are the snap names, verified it with the rest of the documentation. Thanks!

> | Snaps | On the host system, <br/>`/snap/<SNAP_NAME>/current/` </br>Replace <SNAP_NAME> with the name of the Snap.|

Reviewer 3: Not necessarily on the host system. As for appliance the snaps are installed within LXD containers as that is what we target Juju to.

```suggestion
| Snaps | <br/>`/snap/<SNAP_NAME>/current/` </br>Replace <SNAP_NAME> with the name of the Snap.|
```

> | Snaps | On the host system,<br/>`/snap/anbox-cloud/current/COPYRIGHT.*` *for regular Anbox Cloud* <br/>`/snap/anbox-cloud-appliance/current/COPYRIGHT.*` *for Anbox Cloud Appliance*|

Author: Added the generic format for any Snap (based on MM discussion)

> | Snaps | On the host system,<br/>`/snap/$snap-name/current/` </br>Replace $snap-name with the name of the Snap.|

Reviewer 2: I'd avoid using env variable but the angle brackets in the above sentence for clarity.
E.g.
`/snap/<SNAP_NAME>/current/`
(Let's align with what https://snapcraft.io/docs/environment-variables#heading--name doc suggests)

Also this's for the regular deployment. When it comes to appliance, people have to enter one lxd instance to find that out. I'm not quite sure if it's worth mentioning that in the doc though

> | Charms | In the container where the charm is deployed,<br/>`/$path-to-charm/COPYRIGHT` <br/> *$path-to-charm is the path where the charm deployed in the container on the host system* |

Reviewer 3: ```suggestion
| Charms | In the container where the charm is deployed,<br/>`/$path-to-charm/COPYRIGHT` <br/> *$path-to-charm is the path where the charm is deployed. Usually in `/var/lib/juju/agents/unit-*/charm`* |
```
